kt/api-generator/src/main/kotlin/godot/codegen/constants/CoreTypeTraits.kt

```diff
@@ -5,5 +5,6 @@ import godot.codegen.traits.TypedTrait
 object CoreTypeTraits {
     val VECTOR3 = object : TypedTrait {
```

Comment:
More of a question here because it was already there before, but why do we have only one for Vector3 and none of the other core types ?